packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -262,7 +263,7 @@ const AccountTransferNote = ({
                     values={{
                         allowed_internal: addComma(
                             exchange_rate * Number(allowed_transfers_amount?.internal),
-                            2,
+                            is_crypto ? 8 : 2,
```

Comment:
Hi @lubega-deriv,
Can we get the `fractional_digits` from `currency_config`? As the fractional digits might vary for different currencies.

Reply:
good catch, definitely it shouldn't be hardcoded here, 
(unless its done like that in other places, then it still shouldn't be implemented like that but happy to approve given its legacy code anyway)

Reply:
Yep ideally we should use fractional_digits. Updated the implementation 🙏